package spob;

message AckRecover {
  required uint32 primary = 1;
};